packages/flex-dev-utils/src/inquirer.ts

```diff
@@ -1,61 +1,23 @@
 import inquirer, { Question as IQuestion } from 'inquirer';
 import { isSidOfType, SidPrefix } from './sids';
+import { validateConfirmation } from './validators';
 
-export default inquirer;
+// tslint:disable-next-line
```

Comment:
what's wrong with the next line?

Reply:
Same, but I need to remove these ... Why didn't the linter complain about unused variables? 🤔